app/_locales/en/messages.json

```
@@ -272,6 +272,15 @@
   "confirmed": {
     "message": "Confirmed"
   },
+  "confusableUnicode": {
+    "message": "'$1' is similar to '$2'. Be careful!"
+  },
+  "confusableZeroWidthUnicode": {
+    "message": "Zero-width character found. Be careful!"
+  },
+  "confusingEnsDomain": {
+    "message": "This name has some deceptive letters in it that resemble other letters. It is probably trying to impersonate another well-known name. Did you get this name from someone you trust?"
```

Another copy update:

```suggestion
    "message": "We have detected a confusable character in the ENS name. Check the ENS name to avoid a potential scam."
```